components/common/SideBar/index.tsx

```diff
@@ -82,12 +82,21 @@ export default function MiniDrawer() {
             </ListItem>
           </Link>
 
-          <Link href="/finance" passHref>
-            <ListItem button key="Manage Finances">
+          <Link href="/finance/actuals" passHref>
+            <ListItem button key="Manage Actual Finances">
               <ListItemIcon>
                 <InboxIcon />
               </ListItemIcon>
-              <ListItemText primary="Manage Finances" />
+              <ListItemText primary="Manage Actual Finances" />
```

Comment:
the side bar will have only one link to Finance. The default route for finances will redirect to /finance/actuals. From within that page the user can click on the Actual/Plan switch to move between finance/actuals and finance/plan